cli/program_state.rs

```diff
@@ -115,89 +119,69 @@ impl ProgramState {
   /// and traspilation.
   pub async fn prepare_module_load(
     self: &Arc<Self>,
-    module_specifier: ModuleSpecifier,
-    maybe_referrer: Option<ModuleSpecifier>,
+    specifier: ModuleSpecifier,
     target_lib: TargetLib,
     permissions: Permissions,
-    is_dyn_import: bool,
+    is_dynamic: bool,
     maybe_import_map: Option<ImportMap>,
   ) -> Result<(), AnyError> {
-    let module_specifier = module_specifier.clone();
+    let specifier = specifier.clone();
+    let handler = Rc::new(RefCell::new(FetchHandler::new(
+      self,
+      permissions.clone(),
+      self.permissions.clone(),
+    )?));
+    let mut builder = GraphBuilder2::new(handler, maybe_import_map);
+    builder.insert(&specifier, is_dynamic).await?;
```

Comment:
Nitpick: async `insert` method looks strange, maybe we can come up with a better name? `construct()`?

Reply:
But it doesn't construct... it inserts as module in the graph... `.add`?

Reply:
Let's leave it as is for now...